src/MessageBroker/ApacheKafka/KafkaPHP/KafkaPHPAbstractAdapter.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace HelloFresh\Reagieren\MessageBroker\ApacheKafka\KafkaPHP;
+
+use Collections\Dictionary;
+use Collections\MapInterface;
+
+abstract class KafkaPHPAbstractAdapter
+{
+    /**
+     * @var
+     */
+    protected $connection;
+
+    /**
+     * Prevents reconfiguration
+     *
+     * @var bool
+     */
+    protected $configured = false;
+
+    /**
+     * Merge default config with provided configs, and set them on the connection
+     *
+     * @param              $topic
+     * @param MapInterface $configs
+     * @return Dictionary
+     */
+    protected function setConfig($topic, MapInterface $configs)
+    {
+        $configs = (new Dictionary($this->defaults))->concat($configs);
+
+        $this->connection->setGroup($configs->get('group'));
+        $this->connection->setPartition($topic, $configs->get('partition'), $configs->get('offset'));
```

Comment:
@kieranajp Please try to comment this out, and see if you can receive the messages properly.... This was a suggestion that @mandoz did